PubHub/PubHub.API/Domain/PubHubContext.cs

```diff
@@ -89,45 +110,111 @@ protected override void OnModelCreating(ModelBuilder builder)
             {
                 accountType.ConfigureId();
 
+                accountType.Property(at => at.Name)
+                    .HasMaxLength(TYPE_NAME_MAX_LENGTH);
+
                 accountType.TypeToPluralTableName();
             });
 
             builder.Entity<Author>(author =>
             {
                 author.ConfigureId();
 
+                // Author has relations defined in BookAuthor.
+
+                author.Property(a => a.Name)
+                    .HasMaxLength(NAME_MAX_LENGTH);
+
                 author.TypeToPluralTableName();
             });
+            
+            builder.Entity<Book>(book =>
+            {
+                book.ConfigureId();
+
+                book.HasOne(b => b.Publisher)
+                    .WithMany(b => b.Books)
+                    .HasForeignKey(b => b.PublisherId)
+                    .OnDelete(DeleteBehavior.Cascade);
```

Comment:
Do we want to delete an entire publishers library of books when a publisher is removed?

Reply:
Yes, if they are removed for good, I think?? 🤔 That was the thought at least.

Reply:
I'd never use our service 😆 
Imagine buying a book and then having it stripped from your account because the publisher decided to leave a partnership 😮

Reply:
Yeah, I agree. 😂 But wasn't that why we made the relationship between `Publishers` and `Accounts` optional? So that when the publisher leaves (deletes their account), we would still keep their name around in the `Publishers` table?

Reply:
But here we could end up accidently deleting an entire library, which would fuck witha  lot of users.
I think it's a bad idea 🤔

Reply:
Hmm. Should we use `Restrict` instead then? 🤔

Reply:
I believe we should, yes (: